Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:10
+WORKDIR /usr/src/github2range
+COPY package*.json ./
+RUN npm install
+COPY lib/* ./lib/
+CMD ["npm", "start"]
```

Comment:
Looks like this needs an `EXPOSE` line and it would probably be helpful to document what port is being served in the README.

Reply:
This isn't running a server, so no port is needed to be exposed.